src/main/java/com/bandwidth/sqs/consumer/SqsConsumer.java

```diff
@@ -36,54 +34,54 @@
 
 import static com.bandwidth.sqs.consumer.acknowledger.MessageAcknowledger.AckMode;
 
-public class Consumer {
+public class SqsConsumer {
```

Comment:
The `Consumer` class isn't really part of this PR. I'd rather look into this in another PRr.